Plugin gatsby-plugin-remove-generator is not compatible with your gatsby version 5.3.2 - It requires gatsby@2.x.x || 3.x.x || 4.x.x

When using this plugin I get the following warning message:

> Plugin gatsby-plugin-remove-generator is not compatible with your gatsby version 5.3.2 - It requires gatsby@2.x.x || 3.x.x || 4.x.x

> This issue is just about getting the warning message removed

That's only partially true, a dependency tree needs to be updated as well. Currently it doesn't allow `gatsby@5` yet, therefore `npm install` fails. Of course using `--force` or `--legacy-peer-deps` arguments isn't desired and will require updating all our CI configurations.

This is what I get when trying to install _gatsby-plugin-remove-generator_:

> npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: payamha-iran.org-v2@1.0.0
npm ERR! Found: gatsby@5.6.0
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^5.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"2.x.x || 3.x.x || 4.x.x" from gatsby-plugin-remove-generator@1.2.0
npm ERR! node_modules/gatsby-plugin-remove-generator
npm ERR!   gatsby-plugin-remove-generator@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

I'm not so much worried about getting the warning off the way, but rather about updating the dependency tree.

> This issue is just about getting the warning message removed

That's only partially true, a dependency tree needs to be updated as well. Currently it doesn't allow `gatsby@5` yet, therefore `npm install` fails. Of course using `--force` or `--legacy-peer-deps` arguments isn't desired and will require updating all our CI configurations.

This is what I get when trying to install _gatsby-plugin-remove-generator_:

> npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: project-name@1.0.0
npm ERR! Found: gatsby@5.6.0
npm ERR! node_modules/gatsby
npm ERR!   gatsby@"^5.6.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer gatsby@"2.x.x || 3.x.x || 4.x.x" from gatsby-plugin-remove-generator@1.2.0
npm ERR! node_modules/gatsby-plugin-remove-generator
npm ERR!   gatsby-plugin-remove-generator@"*" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.

I'm not so much worried about getting the warning off the way, but rather about updating the dependency tree.

--- a/package.json
+++ b/package.json
@@ -20,7 +20,7 @@
     "gatsby-plugin"
   ],
   "peerDependencies": {
-    "gatsby": "2.x.x || 3.x.x || 4.x.x"
+    "gatsby": "2.x.x || 3.x.x || 4.x.x || 5.x.x"
   },
   "devDependencies": {
     "@commitlint/cli": "^16.2.3",

